app/models/company.rb

```diff
@@ -1,2 +1,7 @@
 class Company < ActiveRecord::Base
+  validates :name,            presence: true
+  validates :description,     presence: true
+  validates :sub_description, presence: true
```

Comment:
![screenshot from 2015-11-06 07 42 53](https://cloud.githubusercontent.com/assets/5815800/10989581/1b26c560-845a-11e5-82ad-bd59e98ee7af.png)

@arafatm From this screen shot, Below the word LOCATION there is address
```
578995, NashVille
```